data/projects/coala_cli.md

```diff
@@ -19,32 +19,84 @@ project is going to rely heavily upon mockups and feedback from the
-details about the project. It is incomplete though and open to a lot of
-improvement.
+[cEP-0010](https://github.com/coala/cEPs/pull/79/files) contains further
+details about the project.
+Make coala easier to use, hence the project is going to rely heavily upon
+mockups and feedback from the community. Adds new functionality so user could
+have multiple options when uses coala. The user interface would help coala
+become much more user friendly and intuitive.
+
 
 #### MILESTONES
 
+[BONDING] https://gitlab.com/coala/GSoC-2017/milestones/16
+[PHASE 1] https://gitlab.com/coala/GSoC-2017/milestones/17
+[PHASE 2] https://gitlab.com/coala/GSoC-2017/milestones/18
+[PHASE 3] https://gitlab.com/coala/GSoC-2017/milestones/19
+
 ##### PREPARATION/BONDING
 
-* The [cEP](https://github.com/coala/cEPs/pull/29/files) has been updated, finished and merged with
+* The [cEP](https://github.com/coala/cEPs/pull/79/files) has been updated, finished and merged with
   the help of mockups and community feedback.
 * A survey is done on the importance of each proposed change to the CLI.
 * Concept for implementation exists in the cEP.
 
+Related issues:
+
+[Google Form] https://gitlab.com/coala/GSoC-2017/issues/54
+[cEP] https://gitlab.com/coala/GSoC-2017/issues/52
+[mockups] https://gitlab.com/coala/GSoC-2017/issues/53
+
+
 ##### CODING PHASE 1
 
 * Implemented at least 3 improvements from the above list including what has
   been proposed by the participant.
 * Thorough testing of the implemented improvements is done and they are merged.
 * Documentation using illustrations of the improvements is merged.
 
+Related issues:
+
+[CLI] https://gitlab.com/coala/GSoC-2017/issues/55
+[Letters instead of numbers] https://gitlab.com/coala/GSoC-2017/issues/127
+[Do nothing] https://gitlab.com/coala/GSoC-2017/issues/130
+[Chaining actions] https://gitlab.com/coala/GSoC-2017/issues/129
+[Testing and writing documentation] https://gitlab.com/coala/GSoC-2017/issues/133
+
 ##### CODING PHASE 2
 
-* Implemented, documented and tested at least 3 more improvements with coding
-  finished and merged.
+* Implement 'Reprompt' for actions
+
+Related issues:
+[Reprompt] https://gitlab.com/coala/GSoC-2017/issues/131
+
+* Implement 'selection action for all results' option
+
+Related issue:
+[selection action] https://gitlab.com/coala/GSoC-2017/issues/132
+
+* Implement 'ApplyPatchAction'
+
+Related issue:
+[apply patch] https://gitlab.com/coala/GSoC-2017/issues/134
 
 ##### CODING PHASE 3
 
 * Implemented, documented and tested all improvements.
+
+Related issue:
+[tests] https://gitlab.com/coala/GSoC-2017/issues/138
+
+* Implement 'interactive patches'
+
+Related issues:
+[patches] https://gitlab.com/coala/GSoC-2017/issues/136
+[functionality] https://gitlab.com/coala/GSoC-2017/issues/137
+
 * Creation of asciinemas showing the usage of the implemented CLI improvements.
+
+Related issue:
+[asciinemas] https://gitlab.com/coala/GSoC-2017/issues/158
+
+
```

Comment:
https://github.com/coala/projects/blob/master/data/projects.js#L376 think you should add the related issues here too